docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/scatter-nd-update-14.rst

```diff
@@ -0,0 +1,230 @@
+.. {#openvino_docs_ops_movement_ScatterNDUpdate_14}
+
+ScatterNDUpdate
+===============
+
+
+.. meta::
+  :description: Learn about ScatterNDUpdate-14 - a data movement operation, which can be 
+                performed on three required input tensors.
+
+**Versioned name**: *ScatterNDUpdate-14*
+
+**Category**: *Data movement*
+
+**Short description**: Creates a copy of the ``data`` input tensor with updates for elements specified by ``indices`` by elements from ``updates`` according to *reduction* attribute.
+
+**Detailed description**: The operation produces a copy of ``data`` tensor and updates its value using logic from ``reduction`` attribute, using values specified
+by ``updates`` at specific index positions specified by ``indices``. The output shape is the same as the shape of ``data``.
+Input ``indices`` can contain duplicated index values, however, in case when *reduction* is set to ``none``, only last update for given duplicated index is used.
```

Comment:
As the previous version of ScatterNDUpdate described behaviour for duplicated indices as "undefined", 
is the usage of "last value of update" logic in fact a common approach in the existing reference and plugins implementation?

Reply:
For existing reference implementation it was the case, however in CPU it was optimized by parallelizing over indices resulting in undefined behavior (last parallel to update the data). For GPU, by looking on code, it might follow reference approach but I'm not sure, need to check it.
If removal of this parallelization would affect performance of models, attribute to switch to "fast but undefined" approach might need to be added